### PR TITLE
Add Mystic style

### PR DESCRIPTION
**Pull request by karst**
Adds Mystic style by @SkyshardMelody.

Need to make sure if the name of the tune is fine or not.

### CODE REVIEW

File: alternative/openrct2.music.mystic/object.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                "source": "music/0.flac",
+                "name": "Mystic Winds",
+                "composer": "Skyshard"
+            }
+        ]
+    },
+    "strings": {
+        "name": {
+            "en-GB": "Mystic style",
+			"nl-NL": "Mystiekstijl"
```
Comment (tupaschoal) [inline alternative/openrct2.music.mystic/object.json]:
```suggestion
            "en-GB": "Mystic style",
			"nl-NL": "Mystiekstijl"
```
Mixed tabs and whitespaces

**Review by tupaschoal — APPROVED** [review]
I like it better than the fantasy theme it is replacing. Not sure about the "mystic" name, but that's a minor thing.

About the author, do they want to keep credits with username?

File: alternative/openrct2.music.mystic/object.json (hunk 1 above)
Comment (karst) [reply alternative/openrct2.music.mystic/object.json]: oh oops, notepad++ acting up again, will fix.

**Comment by karst** [comment]
> I like it better than the fantasy theme it is replacing. Not sure about the "mystic" name, but that's a minor thing.

Would `Mystical` be better? I think it might be. I was looking for a good name for it.

> About the author, do they want to keep credits with username?

They wanted to get credits with their username, instead of real name.

**Comment by IntelOrca** [comment]
I think this is a cross between fantasy and mediveval, but not sure how best to describe that cross-genre.

**Comment by tupaschoal** [comment]
Maybe something like "heroic"?

**Comment by karst** [comment]
Perhaps "Elven" or something like that?

**Comment by tupaschoal** [comment]
I like it

**Comment by tupaschoal** [comment]
I like it

**Comment by Broxzier** [comment]
I don't think "Elves" matches the song well enough, especially at the epic part. I get more of a fantasy, medieval, adventurous, fairy tale vibe from it.

Perhaps "Adventure style" would work?

**Comment by ZehMatt** [comment]
> Perhaps "Elven" or something like that?

Is that even a real genre? I've listened to it and this sounds like some fantasy adventure track.

**Comment by tupaschoal** [comment]
> Is that even a real genre? I've listened to it and this sounds like some fantasy adventure track.

It's trying to replace fantasy, but for some reason I assumed we were not reusing theme names from vanilla.

**Comment by IntelOrca** [comment]
@Broxzier I think Adventure style would be something more like the original Wild West track.

**Comment by karst** [comment]
> @Broxzier I think Adventure style would be something more like the original Wild West track.

I think Frontier style would be better for that. Adventurer's style could be kinda nice tho, kinda that DnD name to it.

**Comment by IntelOrca** [comment]
> could be kinda nice tho, kinda that DnD name to it.

Wouldn't that be campaign style?

**Comment by Broxzier** [comment]
@tupaschoal We're not replacing the original, are we? I still want to use the original one. Or do you mean this would be a fallback track for when the original game isn't linked.

**Comment by karst** [comment]
It's part of the alternative project. For reference, the alternatives are tunes that replace base game music with new songs if the base game isn't installed. Covers replace the masters of old tunes with new (and improved) masters. And additional adds new music without replacing anything.

**Comment by karst** [comment]
Perhaps if we'd call this Elven style we could call the medivial 'Dwarven style'.

**Comment by tupaschoal** [comment]
Naming is the least of our concerns here, we can change it later, don't let it hold you back on merging if you need.

**Pull request merged**